### PR TITLE
Merge GUI tutorial

### PR DESCRIPTION
**Pull request by brenthuisman**
- Merge GUI tutorial.
- Advertise GUI in docs.
- Reduce sidebar clutter by simplying contrib section.

### CODE REVIEW

File: doc/contrib/index.rst [hunk 1]
```diff
@@ -1,127 +1,48 @@
...
-you find an something that fits your expertise or interest, and if
-you’re just getting started, you can look for the “`good 1st
-issue <https://github.com/arbor-sim/arbor/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+1st+issue%22>`__”
-label. Don’t hesitate to post a comment to the Issue if you want to ask
-clarifying questions. If you have implemented a fix, you can make a pull
-request.
-
-Write code
-~~~~~~~~~~
+Arbor uses Git for version control and Github to host its code repository and nearly
```
Comment (thorstenhater) [inline doc/contrib/index.rst:16]:
```suggestion
Arbor uses Git for version control and GitHub to host its code repository and nearly
```

File: doc/contrib/index.rst [hunk 2]
```diff
@@ -1,127 +1,48 @@
...
+all of its infrastructure. If you're not familiar with Git or Github, please have at look at
```
Comment (thorstenhater) [inline doc/contrib/index.rst:17]:
```suggestion
all of its infrastructure. If you're not familiar with Git or GitHub, please have at look at
```

**Review by thorstenhater — CHANGES_REQUESTED** [review]
A styling issue, nothing more. However, are these GUI Screenshots on the latest version?

**Comment by brenthuisman** [comment]
Now with screenshots using the latest version.

**Review by thorstenhater — APPROVED** [review]
Nice! Although that mauve colour :P

**Pull request merged**